src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -1637,6 +1705,21 @@ impl<'a> QemuCmdLine<'a> {
         }
     }
 
+    fn add_scsi_controller(&mut self) {
+        let mut virtio_scsi = DeviceVirtioScsi::new(
+            "scsi0",
+            should_disable_modern(),
+            bus_type(self.config),
+        );
+        if self.config.enable_iothreads {
+            let iothread_id = "scsi-io-thread";
```

Comment:
This assumes that we only declare a single iothread object on the command line. This is currently true but it could change in the future and we'd need a proper allocator.